challenges/functions.js

```diff
@@ -7,36 +7,69 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function sumNums(x, y, cb) {
+  cb(x + y);
+}
+
+sumNums(11, 17, function(total) {
+   console.log(total);
+});
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+function consume(x, y, cb) {
+  cb(x + y);
+}
 
-/* Step 3: Check your work by un-commenting the following calls to consume(): */
-// consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
+consume(1, 1, function(total2) {
+   console.log(total2);
+});
 
+function consume(x, y, cb) {
+  cb(x * y);
+}
 
-// ==== Closures ==== 
+consume(10, 10, function(total3) {
+   console.log(total3);
+});
 
-// Explain in your own words why `nestedfunction()` can access the variable `internal`.
+function consume(firstname, lastname, cb) {
+  cb(firstname + ' ' + lastname);
+}
 
-// Explanation: 
+consume('Asa', 'Shalom', function(total4) {
+   console.log('Hello ' + total4 + ', nice to meet you!');
+});
 
 
-const external = "I'm outside the function";
 
-function myFunction() {
-  console.log(external);
-  const internal = "Hello! I'm inside myFunction!";
+/* Step 3: Check your work by un-commenting the following calls to consume(): */
```

Comment:
Make sure you read all the directions, you consume functions do not console.log so they functions were not written correctly. See if you can fix this.